composer.json

```diff
@@ -41,7 +41,8 @@
         "symfony/yaml": "^2.0.5 || ^3.0 || ^4.0 || ^5.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.35 || ^5.4.3 || ^6.0 || ^7.0"
+        "phpunit/phpunit": "^4.8.35 || ^5.4.3 || ^6.0 || ^7.0 || ^8.0 || ^9.0",
+        "sanmai/phpunit-legacy-adapter": "^6.1 || ^8.0"
```

Comment:
i do understand the need and the approach.
I'm still crying internally.

Reply:
Well, do you suggest we try avoiding PHPUnit 8.0 and stick to 7.0?

That's something we can try, but IIRC there were some difficulties in the coverage collection department.

Reply:
Well, duh.

Reply:
Please never mind my previous comment about PHPUnit 7. I don't know what I was thinking.

 I'll be here for another 10 minutes. If there are other questions or concerns, I can try to address them right here right now.

Reply:
no worry, all good